src/color.js

```diff
@@ -311,30 +311,44 @@ export default class Color {
 
 		if (method.indexOf(".") > 0 && !this.inGamut(space)) {
 			let clipped = color.toGamut({method: "clip", space});
-			if (this.deltaE(clipped, {method: "2000"}) > 2) {
-				// Reduce a coordinate of a certain color space until the color is in gamut
-				let [mapSpace, coordName] = util.parseCoord(method);
+
+			// Reduce a coordinate of a certain color space until the color is in gamut
+			let [mapSpace, coordName] = util.parseCoord(method);
+
+			let delta = "2000";
+			let ε = .01; // for deltaE
+			let JND = 2;
+			if (mapSpace.id === "oklch") {
```

Comment:
```suggestion

			if (mapSpace.id === "oklch") {
```

nit: blank line before control flow blocks